src/applications/static-pages/vre-chapter31/Chapter31CTA.js

```diff
@@ -29,7 +29,7 @@ const Chapter31CTA = props => {
     content = (
       <>
         <a
-          href="/careers-employment/vocational-rehabilitation/apply"
+          href="/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900"
```

Comment:
Is it worth replacing this string with the constant defined in `src/applications/vre/28-1900/constants.js`

Reply:
I agree @Cgfor3 but there is also some ambiguity on the concept of apps importing code from one another. We are kind of cautioned about doing that.

Reply:
It's definitely not a bad idea, but given the length of these CI/CD runs, and my team's desire to have this in the daily deploy, I'll plan on implementing this in a future PR.